### DISCUSSION TITLE
How to configure access control when using Java Client

### DISCUSSION THREAD

**Discussion by zyjnd** [seaweedfs/seaweedfs#5899] [Q&A]
I've configured security.toml for the filer server, using jwt

**Comment by zyjnd** [comment]
@chrislusf

**Reply by chrislusf** [reply]
jwt is not used in java client. only ssl is supported. https://github.com/seaweedfs/seaweedfs/blob/master/other/java/client/src/main/java/seaweedfs/client/FilerSslContext.java#L20

**Comment by zyjnd** [comment]
@chrislusf 
chrislusfHow can i use ssl in java client?and i need use jwt configure when i use ssl?thank your reply

**Comment by zyjnd** [comment]
when i use ssl in java client,will occur a problem(certstrap ,the .crt .key file don‘t set password)
![image](https://github.com/user-attachments/assets/261aa8be-53e0-4a39-a731-d511a55ed9ac)

**Comment by zyjnd** [comment]
i solved,To convert the format of the certificate, convert the PKCS1 format to PKCS8 to convert the certificate format to PKCS8, and convert the PKCS1 format to PKCS8
